storage_service/static/js/project.js

```diff
@@ -1,5 +1,30 @@
 
-
 $(document).ready(function() {
-    $('.datatable').dataTable();
+    $('.datatable').dataTable({
+      // List of language strings from https://datatables.net/reference/option/language
+      oLanguage: {
```

Comment:
Oh I see, that's the way it was done in v1.9? Ok. I upgraded to v1.10 in other places but this is good!